autometa/config/databases.py

```diff
@@ -298,46 +603,87 @@ def configure(self):
 
         Raises
         -------
-        ExceptionName
-            Why the exception is raised.
+        ValueError
+            Provided `section` does not match 'ncbi' or 'markers'.
+        ConnectionError
+            A connection issue occurred when connecting to NCBI or GitHub.
 
         """
-        self.update_missing()
+        self.download_missing(section=section)
+        if no_checksum:
+            return self.config
+        self.fix_invalid_checksums(section=section)
         return self.config
 
 
 def main():
     import argparse
     import logging as logger
 
-    cpus = mp.cpu_count()
     logger.basicConfig(
         format='[%(asctime)s %(levelname)s] %(name)s: %(message)s',
         datefmt='%m/%d/%Y %I:%M:%S %p',
         level=logger.DEBUG)
 
     parser = argparse.ArgumentParser(
-        description='databases config', epilog='By default, with no arguments, will download/format databases into default databases directory.')
+        description='Main script to configure Autometa database dependencies.',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        epilog='By default, with no arguments, will download/format databases into default databases directory.')
+    parser.add_argument('--config',
+        help='</path/to/input/database.config>', default=DEFAULT_FPATH)
+    parser.add_argument('--dryrun',
+        help='Log configuration actions but do not perform them.',
+        action='store_true', default=False)
+    parser.add_argument('--update', help='Update all out-of-date databases.',
+        action='store_true', default=False)
+    parser.add_argument('--update-markers', help='Update out-of-date markers databases.',
+        action='store_true', default=False)
+    parser.add_argument('--update-ncbi', help='Update out-of-date ncbi databases.',
+        action='store_true', default=False)
+    parser.add_argument('--check-dependencies',
+        help='Check database dependencies are satisfied.',
+        action='store_true', default=False)
     parser.add_argument(
-        '--config', help='</path/to/input/database.config>', default=DEFAULT_FPATH)
-    parser.add_argument('--dryrun', help='Log configuration actions but do not perform them.',
-                        action='store_true', default=False)
+        '--no-checksum',
+        help='Do not perform checksum comparisons to validate integrity of database files',
```

Comment:
The description says this is to check the validity of the files. I think this is a little misleading for the user, because it could also trigger if the ftp site has newer databases even though the local files are fine. Perhaps modify the wording?

Reply:
If the related function mentioned in another comment is implemented, then this wording makes a bit more sense.

Reply:
This has been addressed in the new commits